pkg/storegateway/series_chunks_test.go

```diff
@@ -574,6 +589,40 @@ func TestLoadingSeriesChunksSetIterator(t *testing.T) {
 			loadErr:      errors.New("test err"),
 			expectedErr:  "test err",
 		},
+		"loads single set from single block with some ignored histogram chunks": {
+			existingBlocks: []testBlock{block3},
+			setsToLoad: []seriesChunkRefsSet{
+				{series: []seriesChunkRefs{toSeriesChunkRefs(block3, 0), toSeriesChunkRefs(block3, 1)}},
+			},
+			ignoreNativeHistograms: true,
+			expectedSets: []seriesChunksSet{
+				{
+					series: func() []seriesEntry {
+						seriesEntries := make([]seriesEntry, 0, len(block3.series))
+
+						for i := range block3.series[0:2] {
+							refs := make([]chunks.ChunkRef, 0, len(block3.series[i].chks))
```

Comment:
nit: you can move `block3.series[i]` into a variable or use one form the for loop, because `block3.series[i]` repeats 5 times

Reply:
Done!